src/Platform/Vulkan/VkBufferManager.cpp

```
@@ -10,6 +10,8 @@ VkBufferManager::~VkBufferManager() {}
 
 
 Buffer::BufferAlloc VkBufferManager::allocate(VkDeviceSize bufSize, VkBufferUsageFlags usage, Buffer::MemIntent memUsage) {
+	LOG_ASSERT(bufSize > 0, "Cannot allocate an empty (zero-byte) buffer!");
```

Cppcheck (2.20.0)

[information] Include file

(missingIncludeSystem)